jira/client.py

```diff
@@ -2723,11 +2722,11 @@ def delete_user_avatar(self, username, avatar):
         url = self._get_url('user/avatar/' + avatar)
         return self._session.delete(url, params=params)
 
-    def search_users(self, user, startAt=0, maxResults=50, includeActive=True, includeInactive=False):
+    def search_users(self, query, startAt=0, maxResults=50, includeActive=True, includeInactive=False, accountId=None):
```

Comment:
This requires backward incompatible change in the signature, as the problematic parameter is positional.